### ISSUE TITLE
Turning using the Index analog stick sometimes won't register

### ISSUE THREAD

**Issue by leppermessiah1** [Raicuparta/nomai-vr#376]
It goes in and out.

**Comment by Raicuparta** [comment]
Analog stick changes function when holding right hand grip. Make sure you aren't accidentally triggering the grip (can easily happen on index)

**Comment by leppermessiah1** [comment]
> Analog stick changes function when holding right hand grip. Make sure you aren't accidentally triggering the grip (can easily happen on index)

Thanks for the swift response. I'll have to try and see if that helps. I discovered it seemed to be tied to pressing the B button on the right controller, but maybe that's because I was triggering the grip? 

I also found I cannot turn when holding either of the items from my toolbelt. This was true for both Index controllers and Vive wands.

**Comment by Raicuparta** [comment]
> I also found I cannot turn when holding either of the items from my toolbelt. This was true for both Index controllers and Vive wands.

That is intentional.

**Comment by Werschon** [comment]
I also noticed this. It often happens on and around giants deep. Sometimes it recovers after a while (maybe that happens when I accidentally hold an release the grip) but sometimes it doesn't recover and when I go to the pause menu the white selection color always goes to the bottom. It's difficult to explain but when I aim my controller up it moves to the top ('continue') but then it keeps going down as if I'm aiming at the bottom. When this happens it's an indicator for me that my ability to turn won't recover.

I don't know if this has something to do with this issue but that's are the error messages I found in my logs:


<b>[SteamVR]</b> GetOriginTrackedDeviceInfo error (/actions/default/in/LeftHand): InvalidHandle handle: 1152991770271810095 activeOrigin: 0 active: False. Stack trace: | Unity |  
-- | -- | --
<b>[SteamVR]</b> GetOriginTrackedDeviceInfo error (/actions/default/in/RightHand): InvalidHandle handle: 1152991770271810096 activeOrigin: 0 active: False. Stack trace: | Unity |  
[ERROR] An exception occurred inside an event handler or callback. Source: Rewired.ReInput.ControllerConnectedEvent This happens if your event handler/callback code throws an exception. This means the error in your code, not Rewired. Read the exception message and the stack trace carefully to find the source of the exception being thrown by your code. This can also happen if you forget to unsubscribe to an event in a MonoBehaviour class and that object gets destroyed. Make sure you unsubscribe to events in OnDisable or OnDestroy. Rewired will attempt to continue running. Exception: System.NullReferenceException: Object reference not set to an instance of an object at PlayerData.OnSetFirstControllerAfterStartup (Int32 newActiveGamepad, GamePadPresetConfig gamepadConfig, ButtonPromptPresetConfig promptConfig) [0x00000] in <filename unknown>:0 at PadEZ.PadManager_OW.OnReInputControllerConnected (Rewired.ControllerStatusChangedEventArgs obj) [0x00000] in <filename unknown>:0 at Rewired.Utils.SafeAction`1[Rewired.ControllerStatusChangedEventArgs].HMFbhPXIjkKbjeijahJgeZmAyRB (System.Object , System.Action`1 ) [0x00000] in <filename unknown>:0 at Rewired.Utils.SafeDelegate`1[System.Action`1[Rewired.ControllerStatusChangedEventArgs]].Invoke (System.Action`2 invokeCallback) [0x00000] in <filename unknown>:0 ------- Rewired System Info ------- Unity version: 2017.4.33f1 Rewired version: 1.1.35.1.U2017 Platform: Windows Using Unity input: True Primary input source: RawInput Use XInput: True Native mouse handling: True Enhanced device support: True . Stack trace: | Unity |  
<b>[SteamVR]</b> TriggerHapticVibrationAction (/actions/default/out/Haptic) error: InvalidHandle handle: 1152991770271810097. Stack trace: | Unity | 4


Then a after few entries:

Font 'SpaceMono-Regular (UnityEngine.Font)' is not dynamic, which is required to override its style. Stack trace: | Unity | 2
-- | -- | --
Quantum moon orbit position occupied! Aborting collapse.. Stack trace: | Unity |  
NullReferenceException: Object reference not set to an instance of an object. Stack trace: NomaiVR.SmoothFollowCameraRotation.LateUpdate () | Unity | 903
Font 'SpaceMono-Regular (UnityEngine.Font)' is not dynamic, which is required to override its style. Stack trace: | Unity | 6
Fragment 72_GreatSouthernTriangle_Body was moved out of the way before it started moving. Stack trace:


After few more entries:


NullReferenceException: Object reference not set to an instance of an object. Stack trace: NomaiVR.SmoothFollowCameraRotation.LateUpdate () | Unity | 102
-- | -- | --
